Build failed on current version OSX

Brew part is ok:
**1.** `brew install snappy`  
Result:
`==> Pouring snappy-1.1.7_1.high_sierra.bottle.tar.gz`

**2.** git Part is ok  

**3.** `make install -C ldb` its NOT ok  
Result:
`make -C ./deps/leveldb`
`make[1]: *** No targets specified and no makefile found.  Stop.`  
`make: *** [leveldb] Error 2`
Unfortunately I am not a make-scripts expert :-(.
Thx

--- a/Makefile
+++ b/Makefile
@@ -4,7 +4,8 @@ LIBPATH ?= ./lib
 SRC = ldb.cc $(LIBPATH)/*.cc ./deps/docopt/docopt.cpp
 PREFIX ?= /usr/local
 LEVELDBPATH ?= ./deps/leveldb
-LIBLEVELDB ?= $(LEVELDBPATH)/out-static/libleveldb.a
+LEVELDBBUILD ?= ./deps/leveldb/build
+LIBLEVELDB ?= $(LEVELDBBUILD)/libleveldb.a
 LIBSNAPPY ?= ./deps/snappy
 CXXFLAGS += -I$(LEVELDBPATH)/include -std=gnu++11
 
@@ -25,7 +26,8 @@ all: leveldb $(BIN)
 $(LEVELDBPATH):
 	git clone --depth 1 https://github.com/google/leveldb $(LEVELDBPATH)
 leveldb: $(LEVELDBPATH)
-	make -C $(LEVELDBPATH)
+	cmake -DCMAKE_BUILD_TYPE=Release -B$(LEVELDBBUILD) -H$(LEVELDBPATH) && \
+	cmake --build $(LEVELDBBUILD)
 
 $(BIN): $(DEPS) $(LIBPATH)/*.cc deps/docopt
 	$(CXX) -o $(BIN) $(SRC) $(CXXFLAGS) -lpthread -L/usr/local/lib -lsnappy $(LIBLEVELDB) $(DEPS)

--- a/README.md
+++ b/README.md
@@ -8,7 +8,7 @@ A c++ repl and cli tool for leveldb
 ## Mac OS X
 
 ```cli
-$ brew install snappy
+$ brew install snappy cmake
 $ git clone https://github.com/0x00a/ldb.git
 $ make install -C ldb
 ```
@@ -16,7 +16,7 @@ $ make install -C ldb
 ## Linux (Debian / others?)
 
 ```cli
-$ apt-get install libsnappy-dev
+$ apt-get install libsnappy-dev cmake
 $ git clone https://github.com/0x00a/ldb.git
 $ sudo make install
 ```

